### ISSUE TITLE
stable/beta/alpha text bridges misaligned (Zola m.org)

### ISSUE THREAD

**Issue by westerguard** [matrix-org/matrix.org#1719]
Some of the text in the 'class="maturity ...."' is not so well aligned for the bridges. See the print screen below.

![image](https://user-images.githubusercontent.com/92873680/235089197-7ef29463-4ac0-4826-b0fb-b05c54b2b5f2.png)
[Telegram bridge](https://matrix.turbo.fish/ecosystem/bridges/telegram/)

It looks alright for the clients and servers though.
![image](https://user-images.githubusercontent.com/92873680/235090171-38e4331f-dd8d-438c-a44a-795e10eac8b5.png)

**Comment by thibaultamartin** [comment]
Fixed, preview will be deployed in a few minutes

**Comment by HarHarLinks** [comment]
![image](https://user-images.githubusercontent.com/2803622/235605710-b6c911fd-187c-44a0-bf5d-3e443f816212.png)
 personally i'm not quite content yet

**Comment by thibaultamartin** [comment]
Fixed, should be deployed in a few minutes on matrix.turbo.fish

**Comment by HarHarLinks** [comment]
It's improved a bit, but still it looks a little wonky with the larger padding at the bottom for 2 reasons:
- the Stable text looks "too high" within this container
- for probably the same reason, the container looks "too low" compared to the name of the client/bridge/...

@MTRNord says the root cause might actually be the font.

![image](https://github.com/matrix-org/matrix.org/assets/2803622/a3ce910f-e8b6-468a-9019-495807072719)

**Comment by MTRNord** [comment]
Yep just had a talk with @HarHarLinks and it looks like the font has spacing below the visible font. I guess it might be to make it render consistent on the baseline. But it also pushes things here.

**Comment by thibaultamartin** [comment]
Oof ok, I think #1786 is going to fix it, and maybe we should prioritise it before the launch.

**Comment by MTRNord** [comment]
Yeah. Maybe @jplatte has an idea to remove font weirdness at the bottom. Extra space at the top is easier since you could adjust the line height. But I didn't find a workaround for the bottom.

**Comment by HarHarLinks** [comment]
![image](https://github.com/matrix-org/matrix.org/assets/2803622/ebccdb3a-82ec-461c-99aa-4ac3625c03e1)

":left_speech_bubble: Support Room" text and icon look offset from each other too, despite `p`s in the label.
